src/main/java/appeng/tile/misc/TileInscriber.java

```diff
@@ -403,18 +404,28 @@ public IInscriberRecipe getTask()
 				final List<ItemStack> inputs = Lists.newArrayList( startingItem );
 				final InscriberProcessType type = InscriberProcessType.INSCRIBE;
 
-				return new InscriberRecipe( inputs, renamedItem, plateA, plateB, type );
+				final IInscriberRecipeBuilder builder = AEApi.instance().registries().inscriber().builder();
+				return builder.withInputs( inputs )
+						.withOutput( renamedItem )
+						.withTopOptional( plateA )
+						.withBottomOptional( plateB )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Null passed for non-null parameter of appeng.api.features.IInscriberRecipeBuilder.withBottomOptional(ItemStack) in appeng.tile.misc.TileInscriber.getTask() [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=findbugs%3ANP_NULL_PARAM_DEREF)